test/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/rejectedgoodsscheduled/CheckYourAnswersControllerSpec.scala

```diff
@@ -106,10 +109,30 @@ class CheckYourAnswersControllerSpec
     else
       summaryKeys.size shouldBe summaryValues.size
 
-    headers should contain allOf ("Movement Reference Number (MRN)", "Declaration details", "Contact information for this claim", "Basis for claim", "Disposal method", "Details of rejected goods", "Claim total", "Details of inspection", "Supporting documents", "Now send your application")
-
-    summaryKeys should contain allOf ("Contact details", "Contact address", (Seq(
-      "MRN",
+    headers should contain("First Movement Reference Number (MRN)")
+    headers should contain("Declaration details")
```

Comment:
why are these assertions duplicated? I think `should contain allOf` would suffice?

Reply:
The duplication is an oversight. I created individual assertions while debugging so I'll remove the old one.